packages/holocron/src/reactStreaming.js

```diff
@@ -0,0 +1,39 @@
+import { createContext, useContext } from 'react';
+
+const ReactStreamingContext = createContext({});
```

Comment:
I know that there should only be a single instance of holocron but did you think about createReactSharedContext ?

Reply:
I did consider it, but I dont see the point since holocron is externalized. It would just be overkill